### DISCUSSION TITLE
Keyboard enter on menu item selection

### DISCUSSION THREAD

**Discussion by jaymehta-twilio** [twilio-labs/paste#1490] [Q&A] [answered]
Hello team,

We've been using the Menu with actionable menuitems within and noticed that the enter doesn't trigger the onClick action. Was this intentionally left to the application to handle or a bug ? Thanks again.

<img width="1256" alt="Screen Shot 2021-05-25 at 3 36 53 PM" src="https://user-images.githubusercontent.com/52717786/119577364-5b8e3d00-bd6f-11eb-8ebd-6f91607ec6df.png">

**Comment by TheSisb** [comment]
Hi @jaymehta1986,

This looks to be a bug in our menu component. Nice catch and thanks for reporting it!  I've made a ticket in our backlog to fix this. In the meantime keep using the component as-is so the fix will come in easily by just updating the Paste package version.

**Comment by TheSisb** [comment, marked as answer]
The latest version of `@twilio-paste/core` has the fix (v9.3.5)

**Reply by jaymehta-twilio** [reply]
Thank you @TheSisb
